package.json

```diff
@@ -39,7 +32,7 @@
     "coveralls": "^2.11.9"
   },
   "dependencies": {
-    "meow": "^3.7.0"
+    "yargs": "^4.7.1"
```

Comment:
for a couple of reasons we need to use yargs at version 3.30.0 unless there were security/feature fixes that we really need